### DISCUSSION TITLE
Using KIT id in HTML <script> tag

### DISCUSSION THREAD

**Discussion by rogeriokoski** [FortAwesome/Font-Awesome#20364] [General]
If we use the <script> tag adding the CDN for my kit in the <head>  of an HTML (like: <script src="https://kit.fontawesome.com/KIT_ID_NUMBER.js" crossorigin="anonymous"></script>) that gets accessible by every user that access the web page, which in turn can be used and start counting against my page views. This sounds a security exposure and although normal to every application, I think you should make an alert in your docs about using it.

**Comment by robmadole** [comment]
Hi @rogeriokoski 

We have this feature built into Kits:

![THipajAX@2x](https://github.com/user-attachments/assets/2b69501c-583b-4b55-b6cf-0cbe47494aa9)

Does that address your concern?

**Reply by rogeriokoski** [reply]
I think so.
If I go this way, I would refrain from the <script> tag and would use the kit as a package.
Did I get it right ?

Thanks

**Reply by robmadole** [reply]
@rogeriokoski the "Domains" section only applies to the `<script>` tag.

**Reply by rogeriokoski** [reply]
Understood.
I will give it a try

Thanks
